src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -419,7 +419,7 @@
     type: string
     short-summary: The shared access signature for the source storage account.
 examples:
-  - name: Copy multiple blobs to a blob container. Use `az storage blob show` to check the status of the blobs. (autogenerated)
+  - name: Copy multiple blobs or files to a blob container. Use `az storage blob show` to check the status of the blobs. (autogenerated)
```

Comment:
`or files` should be removed. The command only works for blob.

Reply:
manually revert